listener/avalon_listener/tcs_listener.py

```diff
@@ -150,6 +157,21 @@ def parse_command_line(config, args):
             logger.error("Quit : remote_storage_url is not \
                             present in config for Listener")
             sys.exit(-1)
+    if options.zmq_url:
+        if not is_sync:
+            logger.error("Invalid option zmq_url! It should be supported"
+                         "in work order sync mode")
```

Comment:
The wording is confusing. I would say:

```
logger.error("Invalid option zmq_url! It is only supported"
                         "in work order sync mode")
```

Reply:
zmq_url option is supported only in work order sync execution mode. In asynchronous/pull mode this option doesn't make sense and better to throw error.

Reply:
is_sync should be the deciding factor for sync mode ON/OFF. having zmq_url is mandatory to pass if is_sync is true otherwise zmq_url should not have any effect. This way you don't have to modify docker-compose files to modify avalon-listener service to remove --zmq_url if you wish to make is_sync false.

Reply:
currently work order sync mode is set in toml file and it is default option is on. So either case we have to handle this error. Since default is ON so we have to pass this argument.

Reply:
I understand that. But is someone wants to disable the sync, then what will be the procedure to get the docker/standalone builds?

Reply:
Whenever we disable to sync mode then we have to remove those additional parameter, that doesn't make sense.